packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+import {styles as typographyStyles} from "@khanacademy/wonder-blocks-typography";
+
+export default class TextField extends React.Component<{||}> {
+    render(): React.Node {
+        return (
+            <View style={[styles.container, styles.default]}>
+                <input
+                    className={css([
+                        styles.input,
+                        typographyStyles.LabelMedium,
+                    ])}
+                    placeholder="Placeholder"
+                />
+            </View>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
+    container: {
+        width: 288,
```

Comment:
We would prefer leaving this decision to the parent component so we don't add any initial constrains to these fields. The `width` should be set to `100%` and override it if we pass a custom width with the `style` prop.

Reply:
Got it 👍